composer.json

```
@@ -12,17 +12,17 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "laravel/framework": "~5.0",
+        "php": ">=7.2",
```

i think you should revert this change, otherwise laravel this package will break for all users not using Laravel 6
```suggestion
        "php": ">=5.6.0",
```